src/server.js

```diff
@@ -104,6 +127,32 @@ const main = async () => {
     return new Response(200)
   })))
 
+  /**
+   * @api {post} /api/identify Merge two user id's
+   * @apiDescription Calls MixPanel's `$identify` endpoint to merge the included distinct_ids
+   * @apiName event
+   * @apiVersion 1.0.0
+   * @apiGroup Events
+   * @apiParam {Object} properties Properties associated with this event. The below fields are required. Additional application defined fields can also be used
+   * @apiSuccess (Success 200) -
+   */
+  app.post('/api/identify', promiseHandler(withAuth(async req => {
+    validateInput(req.body, Joi.object({ anonId: identifySchema }))
+
+    const data = {
+      event: '$identify',
+      properties: {
+        '$identified_id': userDistinctId(req.user),
+        '$anon_id': req.body.anonId,
+        token
+      }
+    }
+    await Promise.all([
+      token && fetchMixpanel('track', { data })
```

Comment:
Any reason not to also `log` this to stackdriver the way we do above? (If we do decide not to, then we could get rid of the `Promise.all` since there's only one promise)